README.md

```
@@ -104,51 +105,47 @@ example, the JDK distribution ships with a `default.jfc` file containing the top
 
 ```
-CRYOSTAT_AGENT_SMART_TRIGGER_DEFINITIONS="[ProcessCpuLoad>0.2&&TargetDuration>duration(\"1m\")]~default.jfc"
+CRYOSTAT_AGENT_SMART_TRIGGER_DEFINITIONS="{condition:\"ProcessCpuLoad>0.2\",durationExpr:\"1m\",recordingTemplate:\"default.jfc\"}"
 
--Dcryostat.agent.smart-trigger.definitions="[ProcessCpuLoad>0.2&&TargetDuration>duration(\"1m\")]~default.jfc"
+-Dcryostat.agent.smart-trigger.definitions="{condition:\"ProcessCpuLoad>0.2\",durationExpr:\"1m\",recordingTemplate:\"default.jfc\"}"
 ```
 
 Multiple Smart Trigger definitions may be specified and separated by commas, for example:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)